rtos/TARGET_CORTEX/mbed_boot.c

```diff
@@ -712,7 +712,6 @@ void __iar_program_start( void )
     low_level_init_needed_local = __low_level_init();
     if (low_level_init_needed_local) {
         __iar_data_init3();
-    }
```

Comment:
What is an error that this is actually fixing? I don't think this is related to uvisor anyhow.

By removing `}`, the lines 717 and 718 becomes unaligned.